src/test/java/client/PetClient.java

```diff
@@ -15,12 +14,17 @@
 import org.apache.http.HttpStatus;
 import pet.AbstractTest;
 
+import java.io.File;
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.util.*;
 
 public class PetClient extends AbstractTest {
 
+    final private static String PET_PARAMETER = "/v2/pet/";
```

Comment:
Consider renaming these parameters to something like `PET_ENDPOINT`, `FIND_PET_BY_STATUS_ENDPOINT` etc